packages/mobile/package.json

```diff
@@ -143,7 +143,7 @@
     "react-redux": "^7.2.0",
     "redux": "^4.0.5",
     "redux-persist": "^6.0.0",
-    "redux-persist-fs-storage": "^1.3.0",
+    "redux-persist-fs-storage": "git+https://github.com/celo-org/redux-persist-fs-storage#54ffc50",
```

Comment:
Instead of forking this, we could use a yarn resolution to ensure all our dependencies use our patched react-native-fs.
What do you think?

Reply:
or a patch?

Reply:
I didn't know about resolutions, this sounds like the perfect use case for it, I'll do that and delete the fork 😄